de.dlr.sc.virsat.model.extension.statemachines/concept/concept.concept

```diff
@@ -22,7 +22,19 @@ Concept de.dlr.sc.virsat.model.extension.statemachines displayname "State Machin
 	}
 	
 	Category TransitionTriggerEvent description "Some kind of event or action that triggers a transition" {
+		EnumProperty type values [ 

```

Comment:
It can be needed for differentiating between asynchronous (telecommand) and synchronous (auto) behaviour for the verification. Would you substitute it by a StingProperty?

Reply:
I see. Maybe we can call it asychronous and synchronous? Whether that represents a telecommand or a auto(matic?) transition would then be up to the user.